src/module-1/cutStrings/index.js

```diff
@@ -1 +1,6 @@
-export const cutStrings = (arr = []) => {};
+export const cutStrings = (arr = []) => {
+  return arr.map((str) => str.slice(0, arr.reduce((a,b) => a.length <= b.length ? a : b).length));
```

Comment:
Current approach works OK, but has extra complexity, when you search for a min length inside the loop.  
Consider making it simpler, in 2 separate steps:  
1. find the minimum length;  
2. cut each string by that amount.   

*prefer some other array method over `reduce` when seeking the smallest length, for `reduce` is rather a performance-intensive thing (look [Comparing performance of: reduce vs for loop vs forEach](https://www.measurethat.net/Benchmarks/Show/2472/0/arrayreduce-vs-for-loop-vs-arrayforeach))